cmd/manager/main.go

```diff
@@ -166,6 +168,11 @@ func init() {
 		"",
 		fmt.Sprintf("K8s secret mounted into the path designated by %s to be used for webhook certificates", operator.WebhookCertDirFlag),
 	)
+	Cmd.Flags().Bool(
+		operator.DisableAutodiscoveryRBACSetup,
```

Comment:
Nit: I wonder if we should go the opposite way since the other flags are positive (i.e. `EnableAutodiscovery`), so that we can stay consistent between flags

Reply:
I wanted it to be enabled by default, so we would need to modify args in all manifests. Alternatively set the default to true, but this feels off to me. Are we fine with adding another arg?

Reply:
We default `ManageWebhookCertsFlag` to true currently, but I see your point.

>Are we fine with adding another arg?

Do you mean are we fine with adding this arg in particular? I thought we had come to consensus in a previous meeting that we were okay with it, and I still am, but could be persuaded otherwise.